### PR TITLE
[VerifyRemote] Add Kotlin plugin

### PR DESCRIPTION
**Pull request by alexvanyo**
Adds the Kotlin plugin to the VerifyRemote samples, in preparation for modernization.

### CODE REVIEW

**Review by codingjeremy — APPROVED** [review]

File: WearVerifyRemoteApp/Application/build.gradle [hunk 1]
```diff
@@ -1,70 +1,52 @@
...
-
     implementation 'androidx.legacy:legacy-support-v4:1.0.0'
     implementation 'androidx.legacy:legacy-support-v13:1.0.0'
     implementation 'androidx.cardview:cardview:1.0.0'
     implementation 'androidx.appcompat:appcompat:1.2.0'
-
     implementation 'com.google.android.gms:play-services-wearable:17.0.0'
-
     implementation 'com.google.android.support:wearable:2.8.1'
     compileOnly 'com.google.android.wearable:wearable:2.8.1'
```
Comment (codingjeremy) [inline WearVerifyRemoteApp/Application/build.gradle:49]: We are moving to androidx, right? I wasn't sure if this was the first change and then going to androidx?

If you aren't using any of the wearable support library, I would remove these.

File: WearVerifyRemoteApp/Wearable/build.gradle [hunk 2]
```diff
@@ -1,77 +1,50 @@
...
-
     implementation 'androidx.wear:wear:1.1.0'
     implementation 'androidx.wear:wear-phone-interactions:1.0.0-alpha01'
     implementation 'androidx.legacy:legacy-support-v13:1.0.0'
-
-
     implementation 'com.google.android.gms:play-services-wearable:17.0.0'
-
-
     compileOnly 'com.google.android.wearable:wearable:2.8.1'
```
Comment (codingjeremy) [inline WearVerifyRemoteApp/Wearable/build.gradle:48]: Same here.

**Review by codingjeremy — COMMENTED** [review]
Forgot to call out removing wearable support library.

File: WearVerifyRemoteApp/Application/build.gradle (hunk 1 above)
Comment (alexvanyo) [reply WearVerifyRemoteApp/Application/build.gradle:49]: Correct! This was the first change just to add the Kotlin plugin, I'll be removing the support library references in a future PR.

**Pull request merged**